di_website/common/blocks.py

```diff
@@ -111,6 +111,11 @@ class Meta:
         abstract = True
 
 
+class DatasourcesStreamBlock(StreamBlock):
+    link = AbstractLinkBlock()
+    required = False
+
+
```

Comment:
I meant you get rid of this.

Reply:
You're actually not using this at all.

Reply:
Yes `DatasourcesStreamBlock` is unused, just removed it